wk5/5.a/memorygame.js

```diff
@@ -95,9 +97,13 @@ function mousePressed() {
             loop();
         } else {
             gameState.waiting = true;
+            console.log("no match");
             const loopTimeout = window.setTimeout (() => {
+                gameState.flippedCards[0].flip();
```

Comment:
I'm not sure if this is how I did it on the video or not, but in your timeout we needed to:
- flip both of the cards back to face down and then 
- also empty out the flipped cards array so it could be refilled.

After this was added and those few minor variable changes and camelCasing the textSize, everything worked just fine. Oh, and I needed to resize all the images. So I made them all 100 px tall and 60 px wide.